files/jsondata/L10n-Template.json

```
@@ -72,19 +72,22 @@
-    "ja": "この機能は標準ではなく、標準化の予定もありません。公開されているウェブサイトには使用しないでください。ユーザーによっては使用できないことがあります。実装ごとに大きな差があることもあり、将来は振る舞いが変わるかもしれません。",
-    "ko": "이 기능은 비표준이므로 실제 프로덕션에서 사용하지 마세요. 모든 사용자 환경에서 작동하지 않을 수 도 있으며, 미래에 호환성 문제가 생길 수 있습니다.",
-    "ru": "Эта возможность не является стандартной и стандартизировать её пока никто не собирается. Не используйте её на сайтах, смотрящих во внешний мир: она будет работать не у всех пользователей. Также могут присутствовать большие несовместимости между реализациями и её поведение может в будущем измениться.",
-    "zh-CN": "该特性是非标准的，请尽量不要在生产环境中使用它！"
+    "de": "Diese Funktion ist nicht standardisiert. Wir raten davon ab, nicht-standardisierte Funktionen auf produktiven Webseiten zu verwenden, da sie nur begrenzt von Browsern unterstützt werden und sich ändern oder entfernt werden können. In bestimmten Fällen, in denen es keine standardisierte Alternative gibt, kann sie jedoch eine geeignete Option sein.",
+    "en-US": "This feature is not standardized. We do not recommend using non-standard features in production, as they have limited browser support, and may change or be removed. However, they can be a suitable alternative in specific cases where no standard option exists.",
+    "es": "Esta función no está estandarizada. No recomendamos usar funciones no estándar en producción, ya que tienen un soporte limitado en los navegadores y pueden cambiar o eliminarse. Sin embargo, pueden ser una alternativa adecuada en casos específicos donde no exista una opción estándar.",
+    "fr": "Cette fonctionnalité n'est pas standardisée. Nous déconseillons d'utiliser des fonctionnalités non standard en production, car leur prise en charge par les navigateurs est limitée, et elles peuvent être modifiées ou supprimées. Toutefois, elles peuvent constituer une alternative appropriée dans certains cas où aucune option standard n'existe.",
+    "ja": "この機能は標準化されていません。非標準の機能はブラウザーの対応が限られ、将来的に変更または削除される可能性があるため、本番環境での使用は推奨されません。ただし、標準の選択肢が存在しない特定のケースでは、有効な代替手段となる場合があります。",
+    "ko": "이 기능은 표준화되어 있지 않습니다. 비표준 기능은 브라우저 지원이 제한적이며 변경되거나 제거될 수 있으므로 프로덕션 환경에서 사용하는 것은 권장되지 않습니다. 그러나 표준 옵션이 없는 특정 상황에서는 적절한 대안이 될 수 있습니다.",
+    "pt-BR": "Este recurso não é padronizado. Não recomendamos o uso de recursos não padronizados em produção, pois eles têm suporte limitado nos navegadores e podem mudar ou ser removidos. No entanto, podem ser uma alternativa viável em casos específicos onde não exista uma opção padrão.",
+    "ru": "Эта функция не стандартизирована. Мы не рекомендуем использовать нестандартные функции в продакшене, так как поддержка браузерами ограничена, а поведение может измениться или быть удалено. Тем не менее, в некоторых случаях, когда нет стандартного решения, они могут быть подходящей альтернативой.",
+    "zh-CN": "该特性尚未标准化。我们不建议在生产环境中使用非标准特性，因为它们在浏览器中的支持有限，且可能发生变化或被移除。不过，在没有标准选项的特定情况下，它们可能是一个合适的替代方案。",
+    "zh-TW": "此功能尚未標準化。我們不建議在生產環境中使用非標準功能，因為它們的瀏覽器支援有限，且可能變更或被移除。然而，在沒有標準選項的特定情況下，它們可以作為合適的替代方案。"
```

@mdn/yari-content-zh Can you please review these translations by next Thursday (May 22nd)? 🙏

---

LGTM. Thank you @caugner 👍. However, I think the last sentence in Simplified Chinese can use a format similar to the one in Traditional Chinese.

```suggestion
    "zh-CN": "该特性尚未标准化。我们不建议在生产环境中使用非标准特性，因为它们在浏览器中的支持有限，且可能发生变化或被移除。不过，在没有标准选项的特定情况下，它们可以作为合适的替代方案。",
    "zh-TW": "此功能尚未標準化。我們不建議在生產環境中使用非標準功能，因為它們的瀏覽器支援有限，且可能變更或被移除。然而，在沒有標準選項的特定情況下，它們可以作為合適的替代方案。"
```